wpilibc/src/main/native/include/frc/PowerDistribution.h

```diff
@@ -18,7 +18,7 @@ class PowerDistribution : public wpi::Sendable,
                           public wpi::SendableHelper<PowerDistribution> {
  public:
   static constexpr int kDefaultModule = -1;
-  enum class ModuleType { kAutomatic = 0, kCTRE = 1, kRev = 2 };
+  enum class ModuleType { kCTRE = 1, kRev = 2 };
 
   /**
    * Constructs a PowerDistribution.
```

Comment:
Should we say 0 for CTRE, 1 for Rev?

Reply:
This matches what's in the HAL.

Reply:
My question was for farther down in the doxygen comment, where it doesn't allow me to comment.